spyder/plugins/help/widgets.py

```diff
@@ -1094,7 +1094,6 @@ def set_rich_text_font(self, font, fixed_font):
         fixed_font: QFont
             The current rich text font to use.
         """
-
```

Comment:
Removing this space was something you wanted to do, right? Or maybe a missing change to undo after reviewing things here?

Reply:
I wanted to remove it.